_ont/ont-cig-g-97cp.md

```diff
@@ -12,17 +12,17 @@ parent: CIG
 | Vendor/Brand     | CIG              |
 | Model            | G-97CP           |
 | ODM              | ✅               |
-| Chipset          | Realtek RTL9601D |
+| Chipset          | Realtek RTL8672 |
 | Flash            |                  |
 | RAM              |                  |
 | System           | Linux            |
 | 2.5GbaseT        | Yes              |
 | Optics           | SC/APC           |
-| IP address       | 192.168.1.1      |
+| IP address       | 192.168.100.1      |
 | Web Gui          |                  |
-| SSH              |                  |
+| SSH              |✅ user root, password generated with [ZTE F6005 Generator](/ont-zte-f6005)                 |
```

Comment:
Please format the entire table so that the | are vertically aligned (plugins exist for any ide).

Please also add the password generator to the CIG and not link the password generator. Just copy the relevant include.

But are you sure that the ssh password is the same as the enable password?

Reply:
So you still have to enter the generated password twice?

Reply:
Nope, just one time, when you log into the ssh, it gives you the console already enabled, no need to issue the "enable"  and "login" commands. At least for SSH, for serial you need those 2

Reply:
ok top.

Reply:
please specify this, perhaps even in the file that is imported with the password generator.

Reply:
Added in the last commit, feel free to give me feedback to improve it if it doesnt seems clear